testsuite/integration/web/web-enable-elytron.cli

```diff
@@ -1,6 +1,6 @@
 embed-server --server-config=standalone.xml
 
-/subsystem=undertow/application-security-domain=other:add(http-authentication-factory=application-http-authentication)
+#/subsystem=undertow/application-security-domain=other:add(http-authentication-factory=application-http-authentication)
```

Comment:
I would keep it uncommented as we don't want Undertow to touch the legacy for any reason.